Training/interface/DataLoader_main.h

```diff
@@ -1,3 +1,4 @@
+#include "TauMLTools/Analysis/interface/TauTuple.h"
```

Comment:
`TauTuple` code is generated automatically during loading => this include should be removed

Reply:
ok, then let's then substitute the testing script with the ability to test only DataLoader all together (not only c++ part)

Reply:
may I ask just for my understanding, why this include is needed in general?

Reply:
during the development, it was easy to have several module testing steps, one of the script (DataLoader_test.py) was testing only c++ part of the dataloader i.e: before all the functionality that is included in `class DataLoader`, in this testing script compilation of the classes was not included, so I added this, but indeed it may conflict with `makeTree` functionality on the later steps of using dataloader. Now one should use `python DataLoader_fulltest.py` for testing